include/mbedtls/config.h

```diff
@@ -502,16 +507,19 @@
  * function, but will use your mbedtls_internal_ecp_double_jac if the group is
```

Comment:
Minor:
```suggestion
 * function, but will use your mbedtls_internal_ecp_double_jac() if the group is
```
and likewise for other functions so that Doxygen creates hyperlinks.